src/app/modules/gb-cohort-selection-module/constraint-components/gb-constraint/gb-constraint.component.ts

```diff
@@ -48,6 +50,11 @@ export class GbConstraintComponent implements OnInit {
     this.constraintRemoved.emit();
   }
 
+  negate() {
```

Comment:
this method name sounds a bit confusing to me. It updates the constraint negation based on the component slider, right? Is it actually required to save this `excluded` boolean separately, or could the slider also update `constraint.negated` directly?